install/kubernetes/helm/istio-remote/templates/configmap.yaml

```diff
@@ -76,12 +76,26 @@ data:
       # Set concurrency to a specific number to control the number of Proxy worker threads.
       # If set to 0 (default), then start worker thread for each CPU thread/core.
       concurrency: {{ .Values.global.proxy.concurrency }}
-
-    {{- if .Values.global.remoteZipkinAddress }}
```

Comment:
I think this should just use the same `.Values.global.tracer.zipkin.address` - can't think of a good reason for the remote configmap to be different, but might need others who rely on the remote configmap to confirm.

@rshriram  Thoughts?

Reply:
I think this is special because the remote envoy cannot do DNS resolution of zipkin in another cluster (yes, the multicluster thing has lots of holes :( ). So this remoteZipkin is essentially the istio ingress gateway, IIRC.

Reply:
That said, we can fix this later. Just file an issue